src/payment/strategies/paypal-commerce/paypal-commerce-sdk.ts

```diff
@@ -220,28 +221,30 @@ export interface PaypalCommerceHostWindow extends Window {
     paypalLoadScript?(options: PaypalCommerceScriptParams): Promise<{ paypal: PaypalCommerceSDK }>;
 }
 
+export type FundingType = string[];
+export type EnableFundingType =  FundingType | string;
```

Comment:
Is there a reason we are making this looser as it used to be set a set number of strings. (paypal, credit, paylater)

Reply:
Never mind, can see why this is the case 👍